packages/sanity/src/desk/comments/plugin/field/CommentFieldButton.tsx

```diff
@@ -139,37 +147,42 @@ export function CommentFieldButton(props: CommentFieldButtonProps) {
         ref={setPopoverElement}
       >
         <div>
-          <Tooltip disabled={open} portal placement="top" content="Add comment">
-            <Button
-              aria-label="Add comment"
-              disabled={isRunningSetup}
-              icon={AddCommentIcon}
-              mode="bleed"
-              onClick={onClick}
-              size="small"
-              selected={open}
-            />
-          </Tooltip>
+          <Button
+            aria-label="Add comment"
+            disabled={isRunningSetup}
+            icon={AddCommentIcon}
+            mode="bleed"
+            onClick={onClick}
+            size="small"
+            selected={open}
+            tooltipProps={{
+              disabled: open,
+              portal: true,
+              placement: 'top',
+              content: 'Add comment',
+            }}
+          />
         </div>
       </Popover>
     )
   }
 
   return (
-    <Tooltip
-      portal
-      placement="top"
-      content={`View comment${count > 1 ? 's' : ''}`}
-      fallbackPlacements={['bottom']}
-    >
-      <Button aria-label="Open comments" mode="bleed" onClick={onClick} size="small">
+    <Tooltip portal placement="top" content={`View comment${count > 1 ? 's' : ''}`}>
+      <SanityUIButton
```

Comment:
This button has a "special" way of handling children, couldn't reuse the Studio UI Button component